### ISSUE TITLE
Unable to override Cell component with material-ui Tooltip wrapper

### ISSUE THREAD

**Issue by chattling** [material-table-core/core#235]
**Describe the bug**
When I override Cell component and wrapping MTableCell with Tooltip from @material-ui/core, getting Warning in the console.
Also the tooltip eventually doesn't show when hovered over cell.

Override done as follows:
```
components={{
        Cell: (props) => (
          <Tooltip title="My tooltip text">
            <MTableCell {...props} />
          </Tooltip>
        ),
      }}
```

Error:

Warning: Failed prop type: Invalid prop `forwardedRef` of type `function` supplied to `MTableCell`, expected a single ReactElement.
    at MTableCell (http://localhost:3000/uscctools/static/js/8.chunk.js:896:29)
    at MTableCellRef
    at Tooltip (http://localhost:3000/uscctools/static/js/vendors~main.chunk.js:27405:28)
    at WithStyles(ForwardRef(Tooltip)) (http://localhost:3000/uscctools/static/js/vendors~main.chunk.js:37803:31)
    at Cell
    at tr
    at TableRow (http://localhost:3000/uscctools/static/js/1.chunk.js:733:23)
    at WithStyles(ForwardRef(TableRow)) (http://localhost:3000/uscctools/static/js/vendors~main.chunk.js:37803:31)
    at MTableBodyRow (http://localhost:3000/uscctools/static/js/8.chunk.js:4484:37)
    at tbody
    at TableBody (http://localhost:3000/uscctools/static/js/1.chunk.js:233:23)
    at WithStyles(ForwardRef(TableBody)) (http://localhost:3000/uscctools/static/js/vendors~main.chunk.js:37803:31)
    at MTableBody (http://localhost:3000/uscctools/static/js/8.chunk.js:5074:37)

It looks like @material-ui/core is forwarding callback ref (which according to react documentation is possible design pattern: [link to docs](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs), however MTableCell accepts only element:

```
MTableCell.propTypes = {
  columnDef: PropTypes.object.isRequired,
  value: PropTypes.any,
  rowData: PropTypes.object,
  errorState: PropTypes.oneOfType([PropTypes.object, PropTypes.bool]),
  forwardedRef: PropTypes.element,
  size: PropTypes.string,
  colSpan: PropTypes.number,
  children: PropTypes.element,
  cellEditable: PropTypes.bool,
  onCellEditStarted: PropTypes.func
};
```

For contrast - the tooltip works when overriding "Row" component with MTableBodyRow wrapped by Tooltip - no warning and tooltip show when hovered over row.

**Expected behavior**
Expect to work without warning.

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Chrome
- Version:  91.0.4472.114

**Comment by Domino987** [comment]
Thank you for the issue.

This will be fixed with the next release by spreading the props to the cell component.

**Comment by Domino987** [comment]
This will be fixed with the next release: commit 101d290
